languages/elixir/exercises/concept/basic-strings/.docs/after.md

```diff
@@ -0,0 +1,111 @@
+[Strings][getting-started-strings] in Elixir are delimited by double quotes, and they are encoded in UTF-8:
+
+```elixir
+iex> "Hi! ¡Hola! Cześć! Привет!"
+"Hi! ¡Hola! Cześć! Привет!"
+```
+
+Strings can be concatenated using the [`<>/2`][kernel-concat] operator:
+
+```elixir
+iex> "Welcome to" <> " " <> "New York"
+"Welcome to New York"
+```
+
+Strings in Elixir support [interpolation][string-interpolation] using the `#{}` syntax:
+
+```elixir
+iex> "6 * 7 = #{6 * 7}"
+"6 * 7 = 42"
+```
+
+Any Elixir expression is valid inside the interpolation. If a string is given, the string is interpolated as is. If any other value is given, Elixir will attempt to convert it to a string.
+
+```elixir
+iex> "Elixir can convert booleans to strings: #{true}"
+"Elixir can convert booleans to strings: true"
+
+iex> "And #{["lists", ", ", "too!"]}"
+"And lists, too!"
+
+iex> "But not functions: #{fn x -> x end}"
+** (Protocol.UndefinedError) protocol String.Chars not implemented for #Function<7.126501267/1 in :erl_eval.expr/5> of type Function
+    (elixir 1.10.1) lib/string/chars.ex:3: String.Chars.impl_for!/1
+    (elixir 1.10.1) lib/string/chars.ex:22: String.Chars.to_string/1
+```
+
+Elixir provides many functions for working with strings in the _String module_. If you are unsure how to process a string in the way you need it, make sure to browse [functions available in the _String module_][string-module-functions] and you will most likely find what you need.
+
+```elixir
+iex> String.downcase("PLEASE NO SHOUTING")
+"please no shouting"
+
+iex> String.graphemes("cześć")
```

Comment:
Depending on what decision we make for the graphemes, this example should possibly be removed.

Reply:
Changed to a similar example that hides the topic: `String.split("hello", "", trim: true)`.